community/modules/scripts/omnia-install/templates/install_omnia.tpl

```diff
@@ -46,7 +46,7 @@
     ansible.builtin.package:
       name:
       - git
-      state: latest
+      state: present
```

Comment:
FYI: I believe that `state :latest` would not pass Ansible linting if this were a proper Ansible file (vs a template of one). Rationale of change is that `latest` can cause apt or yum to perform potentially slow network operations when you don't really care whether you have the "latest latest" package.